build.gradle

```diff
@@ -15,97 +15,130 @@
  * along with Baritone.  If not, see <https://www.gnu.org/licenses/>.
  */
 
-group 'baritone'
-version '1.2.17'
+allprojects {
+    apply plugin: 'java'
+    apply plugin: "xyz.wagyourtail.unimined"
+    apply plugin: "maven-publish"
+
+    archivesBaseName = rootProject.archives_base_name
+    def vers = 'git describe --always --tags --dirty'.execute().text.trim()
+    if (!vers.startsWith("v")) {
```

Comment:
there was another change in #3486 iirc, but I pulled it in at the beginning of the 1.19 cycle so we could get better error reports

Reply:
#3486 adds some extra logic for pull requests and to make CI use "+" instead of "-" as a separator.
And as of now it is broken because I can't use "'" and '"' properly.

Reply:
As long as leijurv is fine with prs showing an artificial commit in their version (i.e. the reason why #3486 got bloated) I'm also fine with the current handling, except
1. We should use `--match "v*"` to filter version tags (like I suggested in #3486 but without the quotation mistake)
2. This doesn't work without git
```
* What went wrong:
A problem occurred evaluating root project 'baritone'.
> Cannot run program "gitnotinstalled": error=2
```